en/salesforce/src/docs/ms_composer_release_notes_2022.adoc

```diff
@@ -14,6 +14,18 @@ New versions are automatically available once they are released; you don't need
 include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
 
 * You can now transfer binary files as field inputs using data pills from within your Composer flow. For example, when keeping files in sync between Box instances, you can download binary file content from one instance and later upload that same file content to another instance.
+* The `jackson.databind` library is upgraded to version 2.13.4 to address reported security vulnerabilities for HTTP Connector.
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue |ID
+|Fixed the issue in HTTP Connector in which the test connectivity was not triggering a refresh with an OCS connection. |W-11469764
```

Comment:
To avoid the "Fixed the issue in HTTP Connector in which...", you may say:
The test connectivity now triggers a refresh with an OCS connection in HTTP Connector as expected.